Fix website contributing (#221)
beeware/beeware-docs-tools: src/beeware_docs_tools/shared_content/en/contribute/how/dev-environment.md

> {% if not config.extra.website %}

Reviewer: This needs to have a newline after it for tooling reasons. As well, this wraps the content meant to display on the website, so the logic is reversed.

```suggestion
{% if config.extra.website %}

```

> 
> {% if config.extra.website %}
> Contributing to BeeWare requires you to pick a project and set up a development environment:

Reviewer: ```suggestion
{% if config.extra.website %}

Contributing to BeeWare requires you to pick a project and set up a development environment:
```

> Contributing to {{ formal_name }} requires you to pick a project and set up a development environment:

Reviewer: As this is only the website section, it doesn't need to use the Jinja variable syntax. 

```suggestion
Contributing to BeeWare requires you to pick a project and set up a development environment:
```

* [briefcase](https://briefcase.beeware.org/en/stable/how-to/contribute/how/dev-environment/)
* [toga](https://toga.beeware.org/en/stable/how-to/contribute/how/dev-environment/)

> * [rubicon-objc](https://rubicon-objc.beeware.org/en/stable/how-to/contribute/how/dev-environment/)

Reviewer: This should be the proper names.

```suggestion
* [Briefcase](https://briefcase.beeware.org/en/stable/how-to/contribute/how/dev-environment/)
* [Toga](https://toga.beeware.org/en/stable/how-to/contribute/how/dev-environment/)
* [Rubicon Objective-C](https://rubicon-objc.beeware.org/en/stable/how-to/contribute/how/dev-environment/)
```
